tests/dotnet/UnitTests/XcodeProjectTests.cs

```diff
@@ -0,0 +1,378 @@
+			Assert.That (expectedXcodeFxOutput, Does.Exist, $"Expected xcframework output '{expectedXcodeFxOutput}' did not exist.");
+			var outputFxThirdWriteTime = File.GetLastWriteTime (expectedXcodeFxOutput);
+			Assert.IsTrue (outputFxThirdWriteTime > outputFxFirstWriteTime, $"Expected '{outputFxThirdWriteTime}' write time of '{outputFxThirdWriteTime}' to be greater than first write '{outputFxFirstWriteTime}'");
+		}
+
+		[Test]
+		public void BuildMultipleSchemesiOS ()
+		{
+			var platform = ApplePlatform.iOS;
+			Configuration.IgnoreIfIgnoredPlatform (platform);
+
+			var testDir = Cache.CreateTemporaryDirectory (TestName);
+			var proj = Path.Combine (testDir, $"{TestName}.csproj");
+			DotNet.AssertNew (testDir, "iosbinding");
+
+			var xcodeProjName = "TwoSchemeFx";
+			var xcodeProjDirSrc = Path.Combine (XCodeTestProjectDir, xcodeProjName);
+			var xcodeProjDirDest = Cache.CreateTemporaryDirectory ($"{TestName}XcProj");
+			FileHelpers.CopyDirectory (xcodeProjDirSrc, xcodeProjDirDest);
+			AddXcodeProjectItem (proj, Path.Combine (xcodeProjDirDest, $"{xcodeProjName}.xcodeproj"),
+				new Dictionary<string, string> {
+					{ "SchemeName", xcodeProjName },
+				});
+
+			var secondSchemeName = "iOSFx";
+			AddXcodeProjectItem (proj, Path.Combine (xcodeProjDirDest, $"{xcodeProjName}.xcodeproj"),
+				new Dictionary<string, string> {
+					{ "SchemeName", secondSchemeName },
+				});
+
+			DotNet.AssertBuild (proj);
+			AssertXcFrameworkOutput (platform, testDir, xcodeProjName);
+			AssertXcFrameworkOutput (platform, testDir, secondSchemeName);
+		}
+
+		[Test]
+		public void BuildMultipleProjectsiOS ()
+		{
+			var platform = ApplePlatform.iOS;
+			Configuration.IgnoreIfIgnoredPlatform (platform);
+
+			var testDir = Cache.CreateTemporaryDirectory (TestName);
+			var proj = Path.Combine (testDir, $"{TestName}.csproj");
+			DotNet.AssertNew (testDir, "iosbinding");
+
+			var xcodeProjName = "TemplateFx";
+			var xcodeProjDirSrc = Path.Combine (XCodeTestProjectDir, xcodeProjName);
+			var xcodeProjDirDest = Cache.CreateTemporaryDirectory ($"{TestName}XcProj");
+			FileHelpers.CopyDirectory (xcodeProjDirSrc, xcodeProjDirDest);
+			AddXcodeProjectItem (proj, Path.Combine (xcodeProjDirDest, $"{xcodeProjName}.xcodeproj"),
+				new Dictionary<string, string> {
+					{ "SchemeName", xcodeProjName },
+				});
+
+			var xcodeProjName2 = "TwoSchemeFx";
+			var xcodeProjDirSrc2 = Path.Combine (XCodeTestProjectDir, xcodeProjName2);
+			var xcodeProjDirDest2 = Cache.CreateTemporaryDirectory ($"{TestName}XcProjTwo");
+			FileHelpers.CopyDirectory (xcodeProjDirSrc2, xcodeProjDirDest2);
+			AddXcodeProjectItem (proj, Path.Combine (xcodeProjDirDest2, $"{xcodeProjName2}.xcodeproj"),
+				new Dictionary<string, string> {
+					{ "SchemeName", xcodeProjName2 },
+				});
+
+			DotNet.AssertBuild (proj);
+			AssertXcFrameworkOutput (platform, testDir, xcodeProjName);
+			AssertXcFrameworkOutput (platform, testDir, xcodeProjName2);
+		}
+
+		[Test]
```

Comment:
Adding the "Multiplatform" is needed for tests that require multiple platforms, otherwise they won't be run on CI (because we run tests in parallel per-(single-)platform basis, and if a test needs multiple platforms enabled, none of the single-platform variations will run it. So we have a multi-platform configuration, but tests must opt into it by adding the "Multiplatform" category):

```suggestion
		[Test]
		[Category ("Multiplatform")]
```

Reply:
Thanks, added